### PR TITLE
update /healthcheck to note the build commit - Issue 2181

### LINKED ISSUE #2181: update /healthcheck to note the build commit rather than version
we're not versioning thimble in a meaningful way: we should update healthcheck so that it lists its current commit instead, so that we actually know which version of Thimble we're looking at on staging or production.

### PR DESCRIPTION
**Pull request by ericsuhn**
This is for fix #2181 using [git-rev-sync](https://github.com/kurttheviking/git-rev-sync-js) to record the current commit in /healthcheck.

### CODE REVIEW

**Comment by humphd** [comment]
@gideonthomas can you review this one please?

File: server/request.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ let cookieParser = require("cookie-parser");
 let cookieSession = require("cookie-session");
 
 let version = require("../package").version;
+let commit = require("../node_modules/git-rev-sync/").long();
```
Comment (humphd) [inline server/request.js]: You don't need to use `../node_modules/`, just do:

```js
let gitRevSync = require("git-rev-sync").long();
```

File: services/login.webmaker.org/app/http/controllers/site.js [hunk 2]
```diff
@@ -38,14 +38,16 @@ exports.js = function (filename) {
  * GET health check for app
  */
 var version = require("../../../package").version;
+var commit = require("../../../node_modules/git-rev-sync/").long;
```
Comment (humphd) [inline services/login.webmaker.org/app/http/controllers/site.js]: Same thing here, no pathing to get to `git-rev-sync`

Comment (gideonthomas) [reply services/login.webmaker.org/app/http/controllers/site.js]: undo changes in this file...we shouldn't update services in here.

File: server/request.js (hunk 1 above)
Comment (gideonthomas) [reply server/request.js]: +1 to not using "node_modules" in the path. But let's call `long()` only once.
So, it should look like: 
```js
const commit = require("git-rev-sync").long();
...
res.json({ http: "okay", version, commit });
```

**Comment by ericsuhn** [comment]
Thanks for the review! I've committed the requested changes.

File: server/request.js [hunk 3]
```diff
@@ -7,6 +7,7 @@ let cookieParser = require("cookie-parser");
 let cookieSession = require("cookie-session");
 
 let version = require("../package").version;
+const gitRevSync = require("git-rev-sync").long();
```
Comment (gideonthomas) [inline server/request.js]: @ericsuhn can you rename this to `commit`...then I think this should be good to land

**Comment by ericsuhn** [comment]
done, thanks @gideonthomas

**Comment by gideonthomas** [comment]
Thanks @ericsuhn! This looks great

**Pull request merged**